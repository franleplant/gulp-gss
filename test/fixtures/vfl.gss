@horizontal [#find]-[#findNext]-[#findField] chain-top;
