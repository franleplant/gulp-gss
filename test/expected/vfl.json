{
  "selectors": [
    "#find",
    "#findNext",
    "#findField"
  ],
  "vars": [
    [
      "get",
      "#find[right]",
      "right",
      [
        "$id",
        "find"
      ]
    ],
    [
      "get",
      "[hgap]",
      "hgap"
    ],
    [
      "get",
      "#findNext[left]",
      "left",
      [
        "$id",
        "findNext"
      ]
    ],
    [
      "get",
      "#findNext[right]",
      "right",
      [
        "$id",
        "findNext"
      ]
    ],
    [
      "get",
      "#findField[left]",
      "left",
      [
        "$id",
        "findField"
      ]
    ],
    [
      "get",
      "#find[top]",
      "top",
      [
        "$id",
        "find"
      ]
    ],
    [
      "get",
      "#findNext[top]",
      "top",
      [
        "$id",
        "findNext"
      ]
    ],
    [
      "get",
      "#findField[top]",
      "top",
      [
        "$id",
        "findField"
      ]
    ]
  ],
  "constraints": [
    [
      "eq",
      [
        "plus",
        [
          "get",
          "#find[right]"
        ],
        [
          "get",
          "[hgap]"
        ]
      ],
      [
        "get",
        "#findNext[left]"
      ]
    ],
    [
      "eq",
      [
        "plus",
        [
          "get",
          "#findNext[right]"
        ],
        [
          "get",
          "[hgap]"
        ]
      ],
      [
        "get",
        "#findField[left]"
      ]
    ],
    [
      "eq",
      [
        "get",
        "#find[top]"
      ],
      [
        "get",
        "#findNext[top]"
      ]
    ],
    [
      "eq",
      [
        "get",
        "#findNext[top]"
      ],
      [
        "get",
        "#findField[top]"
      ]
    ]
  ],
  "css": ""
}
